src/libs/ReportUtils.js

```diff
@@ -1508,15 +1508,17 @@ function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceip
     }
 
     if (shouldConsiderReceiptBeingScanned && ReportActionsUtils.isReportPreviewAction(reportAction)) {
-        const linkedTransaction = TransactionUtils.getTransaction(reportAction.childLastReceiptTransactionIDs);
+        const lastTransactionID = reportAction.childLastReceiptTransactionIDs.split(',')[0];
```

Comment:
When there are many transactions in a report, childLastReceiptTransactionIDs will be "1231245654,123454222,1234545344,12346654"

because this logic
https://github.com/Expensify/App/blob/109e96c982352eea8862ddf08b14b29d2c45e56d/src/libs/ReportUtils.js#L2435

And in this case, we need to get the last transaction